README.md

```diff
@@ -16,32 +16,51 @@ environment:
 configmap:
   VAR_A: VALUE_A
   VAR_B: VALUE_B
+secrets: 
+  ENVIRONMENT_VAR:
+      secretRef: some-secret-reference
+      key: connectionString
 ```
 
+### secrets
+Secret objects in this configuration consist of name secretRef and key. These need to be put into the secrets array in the following manor
+```yaml
+secrets: 
+  ENVIRONMENT_VAR:
+      secretRef: some-secret-reference
+      key: connectionString
+```
+*Where:*
+- **ENVIRONMENT_VAR** is the environment variable name to reference ( this has to be capitals and can contain numbers or "_" ).
+- **secretRef** is the service instance ( as in the case of PaaS wrappers ) or reference to the secret volume
+- **key** is the 
```

Comment:
did you forget to finish this sentence? :-)

Reply:
yup ill tidy that up too.